mix.exs

```diff
@@ -6,21 +6,24 @@ defmodule SocketProxy.Mixfile do
       app: :socket_proxy,
       version: "0.1.0",
       elixir: "~> 1.9",
-      start_permanent: Mix.env == :prod,
+      start_permanent: Mix.env() == :prod,
       deps: deps(),
       releases: [
         socket_proxy: [
           include_executables_for: [:windows],
           applications: [runtime_tools: :permanent]
         ]
+      ],
+      dialyzer: [
+        plt_add_apps: [:mix]
       ]
     ]
   end
 
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:logger],
+      extra_applications: [:logger, :ranch],
```

Comment:
I don't think you need to list it here. In ye old days of Elixir, when this was an `applications:` key you did, but now, it automatically includes the things in `deps`, and you only need to list `extra_applications` that don't appear there (primarily `:logger`, and some erlang built-ins, I think).

Reply:
Ah, okay... I noticed that it _did_ work without this, but it was included in one of the old "how to use Ranch from Elixir" blog posts I was looking at, so I included it to be safe. I'll take it out.